backend/tests/integration_docker/test_triggered_actions.py

```diff
@@ -0,0 +1,195 @@
+import asyncio
+from pathlib import Path
+
+import pytest
+from infrahub_sdk import InfrahubClient
+from infrahub_sdk.protocols import CoreGenericRepository
+from infrahub_sdk.schema import NodeSchema, SchemaRoot
+from infrahub_sdk.testing.docker import TestInfrahubDockerClient
+from infrahub_sdk.testing.repository import GitRepo
+from infrahub_sdk.testing.schemas.car_person import (
+    TESTING_CAR,
+    TESTING_MANUFACTURER,
+    TESTING_PERSON,
+    SchemaCarPerson,
+)
+from prefect.client.orchestration import PrefectClient
+
+from infrahub.core.constants import InfrahubKind
+from infrahub.trigger.constants import NAME_SEPARATOR
+from tests.helpers.fixtures import get_fixtures_dir
+
+CURRENT_DIRECTORY = Path(__file__).parent.resolve()
+
+
+class TestTriggeredActions(TestInfrahubDockerClient, SchemaCarPerson):
+    @pytest.fixture(scope="class")
+    def infrahub_version(self) -> str:
+        return "local"
+
+    @pytest.fixture(scope="class")
+    def schema_person_artifact(self, schema_person_base: NodeSchema) -> NodeSchema:
+        person_schema = schema_person_base.model_copy(deep=True)
+        person_schema.inherit_from = [InfrahubKind.ARTIFACTTARGET]
+        return person_schema
+
+    @pytest.fixture(scope="class")
+    def initial_schema(
+        self,
+        schema_car_base: NodeSchema,
+        schema_person_artifact: NodeSchema,
+        schema_manufacturer_base: NodeSchema,
+    ) -> SchemaRoot:
+        return SchemaRoot(
+            version="1.0",
+            nodes=[schema_person_artifact, schema_car_base, schema_manufacturer_base],
+        )
+
+    async def test_load_initial_schema(
```

Comment:
Do following tests rely on this test having ran prior to them? If so, this test should be either a fixture or belongs to the test setup, even if we may currently have this pattern within `test_propose_change_repository`

Reply:
Good point, I've converted those into fixtures.